docker/sys/Dockerfile.caddy

```diff
@@ -0,0 +1,9 @@
+FROM node
+WORKDIR /usr/src/app
+ADD client/ .
+RUN npm install
+RUN npm run build
+
+FROM abiosoft/caddy
```

Comment:
Its a temp build step. We build the UI in its own container, snag the files from that container, and then throw out the container because we don't need it anymore